src/app/components/pagination/pagination.component.ts

```diff
@@ -64,117 +64,136 @@ export class PaginationComponent implements OnInit, OnChanges {
   }
 
   onClickPrevButton(): void {
-    const selectedTodos: number[] = [];
-
-    this.lastPage = this.todosLength -  this.countClick; /*остаток с последней страницы*/
-
-    if (this.countClick >= 3) {
-      this.countClick -= 3; /*если нажали то добаляем 3 */
-    }
-    this.numberButtonClick -= 1;
-
-    if (this.todosLength >  this.countClick) {
-      selectedTodos.length = 0;
-      for (let i = this.countClick ; i < this.countClick + 3; i++ ) {
-        selectedTodos.push(i);
-      }
-    }
-    /*если осталось 1 елемента*/
-    if (this.lastPage === 1) {
-      selectedTodos.length = 0;
-      for (let i = this.todosLength - 4; i < this.todosLength - 1; i++ ) {
-        selectedTodos.push(i);
-      }
-    }
-    /*елси остался 2 елемент*/
-    if (this.lastPage === 2) {
-      selectedTodos.length = 0;
-      for (let i = this.todosLength - 5; i < this.todosLength - 2; i++ ) {
-        selectedTodos.push(i);
-      }
-    }
-
-    if (this.countClick <= 0) {
-      selectedTodos.length = 0;
-      this.countClick = 0;
-      this.selectNumberButton = 1;
-      for (let i = this.countClick; i <  this.countClick + 3; i++ ) {
-        selectedTodos.push(i);
-      }
-    }
-
-    const from: number = selectedTodos[0];
-    const to: number = selectedTodos[2];
-    this.clickPrevButton.emit([from, to]);
+    this.numberButtonClick -= 1
+  
+    var to = numberButtonClick * countElementsInPage;
+    var from = to - countElementsInPage + 1
+    
+     this.clickPrevButton.emit([to, from]);
```

Comment:
export inteface paginationConfigInterface {
from: number;
to: number:
}

{
from: 7,
to: 9
}